Tests/kaas/k8s-version-policy/k8s_version_policy.py

```diff
@@ -474,11 +555,23 @@ async def main(argv):
         logger.critical("The EOL data in %s is outdated and we cannot reliably run this script.", EOLDATA_FILE)
         return 1
 
+    kubeconfig_path = config.kubeconfig
+
     connector = aiohttp.TCPConnector(limit=5)
     async with aiohttp.ClientSession(connector=connector) as session:
         cve_affected_ranges = await collect_cve_versions(session)
     releases_data = fetch_k8s_releases_data()
 
+    try:
+        logger.info(f"Checking cluster specified by {kubeconfig_path}")
+        cluster = await get_k8s_cluster_info(config.kubeconfig, config.context)
+        await scan_k8s_images(cluster.kubeconfig)
```

Comment:
I have the impression that `config.kubeconfig` is the same as `cluster.kubeconfig`. Why did you introduce the new field for `ClusterInfo`??